### PR TITLE
Add noise mitigation cuts for SR in VBF

### PR DESCRIPTION
**Pull request by alpakpinar**
Quick list of changes in this PR:
- Smearing turned off for VBF
- Noise mitigation cuts based on VecB/VecDPhi and neutral EM energy fractions are added in VBF SR
- Plotting updates in PR #240 are also here

### CODE REVIEW

**Comment by AndreasAlbert** [comment]
hey @alpakpinar apologies for not merging this already. Should I still merge?

File: bucoffea/plot/stack_plot.py [hunk 1]
```diff
@@ -290,15 +303,15 @@ def make_plot(acc, region, distribution, year,  data, mc, signal=None, outdir='.
...
-    fig.text(1., 1., f'{channel_name(region)}, {lumi(year)} fb$^{{-1}}$ ({year})',
+    fig.text(1., 1., f'{channel}, {lumi(year)} fb$^{{-1}}$ ({year})',
                 fontsize=14,
                 horizontalalignment='right',
                 verticalalignment='bottom',
                 transform=ax.transAxes
                )
     # Aesthetics
     ax.set_yscale("log")
-    ax.set_ylabel('Events / GeV')
```
Comment (AndreasAlbert) [inline bucoffea/plot/stack_plot.py]: just noticed this small thing: please keep the 'Events / GeV' label

**Comment by alpakpinar** [comment]
Hey @AndreasAlbert , I changed the y-label in stack plots to "Events / GeV", also pushed a second merge commit for fixing some conflicts with master branch. (Because of the merge and the inclusion of deepTau, now the PR tests fail I believe, should we maybe update the testing scripts to v7? Or maybe include a similar handle with photons to handle both v5 and v7 tau ID?) Thanks!

**Comment by AndreasAlbert** [comment]
I should really just update the testing files, which are still v5 :) Thanks for the update, I'll merge now

**Comment by alpakpinar** [comment]
Ah yeah sorry that's what I wanted to say :) Thanks a lot! I just had a quick comment on this PR: This PR contains all the plotting updates in PR #240 already, so once this is merged, I don't think we need to merge that one. Thanks!

**Pull request merged**